src/factories/createRoarrInitialGlobalState.ts

```diff
@@ -13,14 +13,16 @@ export default (currentState: any): RoarrGlobalState => {
     versions.sort(cmp);
   }
 
-  const currentIsLatestVersion = !versions.length || cmp(pkg.version, versions[versions.length - 1]) === 1;
+  const currentIsLatestVersion = versions.length === 1 || cmp(pkg.version, versions[versions.length - 1]) === 1;
 
-  if (!versions.includes(pkg.version)) {
+  if (pkg.version && !versions.includes(pkg.version)) {
     versions.push(pkg.version);
+    
+    if (versions.length > 1) {
```

Comment:
@gajus I moved inside because I understand is not necessary to sort the array because it's sorted since the last time.